saleor/graphql/order/utils.py

```diff
@@ -98,8 +128,12 @@ def validate_draft_order(order, country):
 
     Returns a list of errors if any were found.
     """
+    validate_billing_address(order)
     if order.is_shipping_required():
-        validate_shipping_method(order)
+        validate_shipping_address(order)
```

Comment:
Not sure of this `if..else` 
If the order doesn't require shipping, then why do we validate shipping_method?

Reply:
@tomaszszymanski129  IMHO, the parameter `required` is not needed. this `if else` is also not needed.
```
if order.is_shipping_required():
    validate_shipping_address(order)
    validate_shipping_method(order)
```
If order requires the shipping then we will validate it.